src/components/DiagonalGradient.js

```diff
@@ -4,16 +4,22 @@ import { LinearGradient } from 'expo';
 
 import { colors } from '../config';
 
-export const DiagonalGradient = ({ children }) => (
-  <LinearGradient
-    colors={[colors.primary, colors.secondary]}
-    start={{ x: 0, y: 0 }}
-    end={{ x: 1, y: 1 }}
-  >
-    {children}
-  </LinearGradient>
-);
+const defaultProps = {
```

Comment:
maybe we can use the components default props, which would be the more typical way. I will try that out:

```
DiagonalGradient.defaultProps = {
  colors={[colors.primary, colors.secondary]}
  start={{ x: 0, y: 0 }}
  end={{ x: 1, y: 1 }}	  
  style: { flex: 1 }
};
```

than we do not have do set it explicitly here, because the PropTypes logic knows what to do:

```
<LinearGradient {...props}>
```